src/lib/features/feature-search/feature-search-controller.ts

```diff
@@ -56,6 +55,8 @@ export default class FeatureSearchController extends Controller {
                     summary: 'Search and filter features',
                     description: 'Search and filter by selected fields.',
                     operationId: 'searchFeatures',
+                    // TODO: fix the type
+                    parameters: featureSearchQueryParameters as any,
```

Comment:
@thomasheartman can you help here? Do we have a standard method for externalizing query params to a type without type assertions?